lib/OpenQA/CacheService/Response/Info.pm

```diff
@@ -25,10 +25,18 @@ sub available_workers {
     return $data->{active_workers} != 0 || $data->{inactive_workers} != 0;
 }
 
+sub inactive_jobs_exceeded {
+    my $self = shift;
```

Comment:
It would be nice to use signatures in new code, e.g.
```suggestion
sub inactive_jobs_exceeded ($self) {
```

Reply:
sure - if that works